src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3544,6 +3544,28 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    [ValueConversion(typeof(PortType), typeof(string))]
```

Comment:
Add some comments to a public converter.

Reply:
Yup!